python3/httplib2/__init__.py

```diff
@@ -185,16 +185,15 @@ def _get_end2end_headers(response):
 
 def _errno_from_exception(e):
```

Comment:
```suggestion
def _errno_from_exception(e):
    errno = getattr(e, "errno", None)
    if errno is not None:
        return errno

```

i think this is required to avoid traversing E.args[0].args[0].... when errno is already available in upper layer